src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -33,7 +34,8 @@ const AuthContainer: FunctionComponent = ({ children }) => {
       className={classes}
       data-version={VERSION}
       data-commit={COMMITHASH}
-      lang={getLanguageCode(widgetProps)}
+      lang={lang}
+      dir={dir}
```

Comment:
Actually, wondering if we should pull from context rather than passed as a prop. Thoughts?